### DISCUSSION TITLE
Version 6.1.0 prerelease 22070602 online

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#3011] [Announcements]
I put a prerelease of v6.1.0 online in NuGet.

You can track the release progress in issue #3009 

Version 6.1.0 adds support for Maui and also includes several enhancements and bug fixes.

The most important bug fix is that `LocalProxy` now properly disposes scoped services (like your database connection) as expected.

The one potential breaking change is that Blazor wasm no longer requires text serialization, and so the default behavior has been changed to not use text serialization. This means that **your data portal server** needs to be updated to no longer require text serialization.

Please let me know if you encounter any issues with this new version. I expect it to go live in about a week if no issues are reported.

**Comment by hoangcuongvn** [comment]
Please add supporting parameters for `Execute` and `ExecuteAsync`.
https://github.com/MarimerLLC/csla/discussions/2923#discussioncomment-2801136

**Reply by rockfordlhotka** [reply]
That'll be in the next release. I cut out a bunch of things from 6.1.0 to get it out sooner, because the change to `LocalProxy` to properly dispose the scope is a critical fix.

**Comment by hoangcuongvn** [comment]
I think there is a small bug in `ChildDataPortal`. The code line below should be `obj.Child_OnDataPortalInvokeComplete(eventArgs);`.

https://github.com/MarimerLLC/csla/blob/852887592f7bf554ccb0aa4ee1d67feb766b05a7/Source/Csla/Server/ChildDataPortal.cs#L99

**Reply by rockfordlhotka** [reply]
Thank you @hoangcuongvn
